commit 5f96c989bc3cc7e0fcd499700e174fd6bb3c8897
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-07-11T16:53:27-07:00 (3 months ago)
    
    joyent/node-sshpk-agent#19 update jsstyle submodule so make check works again

diff --git a/deps/javascriptlint b/deps/javascriptlint
index 040bf5e..ad52812 160000
--- a/deps/javascriptlint
+++ b/deps/javascriptlint
@@ -1 +1 @@
-Subproject commit 040bf5e429969ae42d976b570fd64c9e17fab20f
+Subproject commit ad52812e77bdfb1e90fb71a1201adb2b665a27e6
diff --git a/deps/jsstyle b/deps/jsstyle
index 194dde7..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit 194dde793f58b707b0416a106e1f7679795e50df
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
